server/notification-providers/zoho-cliq.js

```diff
@@ -84,7 +84,7 @@ class ZohoCliq extends NotificationProvider {
 
             const payload = this._notificationPayloadFactory({
                 monitorMessage: heartbeatJSON.msg,
-                monitorName: monitorJSON.name,
+                monitorName: monitorJSON.pathName,
```

Comment:
no idea if this would break something

```suggestion
                monitorName: monitorJSON.name,
```